src/main/kotlin/net/atos/zac/app/admin/model/RESTBuildInformation.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos, 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.app.admin.model
+
+import java.time.LocalDateTime
+
+data class RESTBuildInformation(
+    val commit: String?,
```

Comment:
```suggestion
    val commit: String,
```

? Same with the other fields. Or is there a reason why they are nullable?

Reply:
In local / dev scenario we don't have this info, so decided to leave them nullable

Reply:
I can confirm that these are null/missing when in DEV / IntelliJ mode